### DISCUSSION TITLE
How to know when heavy GLTF model fully loaded in sceneView ?

### DISCUSSION THREAD

**Discussion by agGitHub** [sceneview/sceneform-android#246] [Q&A] [answered]
Hi,

I have a heavy GLTF model. It needs about 5 seconds to load. I would like to have a callback in order to know when the loading of  all the textures of the scene/sceneview is finished.

Does this exist? Or any way to detect this event?

Thanks

**Comment by grassydragon** [comment, marked as answer]
Hi!
I think, if you don't use the asynchronous texture loading, the textures will be loaded before you receive the callback with the renderable: https://github.com/SceneView/sceneform-android/blob/master/core/src/main/java/com/google/ar/sceneform/rendering/Renderable.java#L425
